pkg/internal/testhelpers/containers.go

```diff
@@ -36,6 +36,8 @@ const (
 
 	Superuser   = true
 	NoSuperuser = false
+
+	LatestDBWithPromscaleImageBase = "cevian/promscale-extension"
```

Comment:
This needs to change before merge but allows the CI to test using the new extension